sasdata/dataloader/readers/anton_paar_saxs_reader.py

```diff
@@ -62,7 +62,7 @@ def get_file_contents(self):
 
         # Reinitialize the class when loading a new data file to reset all class variables
         self.reset_state()
-        buff = self.readall()
+        buff = self.readall().strip()
```

Comment:
This only strips white space from the start and end of the text, not for each line. Are you sure you don't want to handle blank lines/comment lines as part of the normal line-by-line processing of raw data?

Using the `raw_data` attribute to hand data to `read_data` is a bit odd. With a name like `read_data` I would expect it to read the data, but at that point the data has already been read. But that's outside the scope of this PR.